src/styles.css

```diff
@@ -10,23 +10,33 @@
   border: 2px black solid;
 }
 
+.main {
+  display: flex;
+  flex-direction: column;
+}
+
```

Comment:
Lets try working these dimensions in @media screen { } rule and then style for the laptop size above that.